### ISSUE TITLE
running xeger.Generate() causes stack overflow exception

### ISSUE THREAD

**Issue by mohamadbahmani** [moodmosaic/Fare#57]
string regexPattern=@"^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]{8,}$"

```
public static string GenerateRandomStringFromRegex(string regexPattern)
        {
            var xeger = new Xeger(
            return xeger.Generate();
        }
```

**Comment by gregsdennis** [comment]
Maybe a duplicate of #5?

**Comment by moodmosaic** [comment]
Thank you for reporting this. It can be resolved as part of #5, so we can close this issue and continue the discussion there. Feel free to re-open if/when needed. 👍
